src/pivot-table/components/cells/DimensionCell.tsx

```diff
@@ -44,11 +44,21 @@ const stickyCell: React.CSSProperties = {
   top: 4,
 };
 
-const dimTextStyle: React.CSSProperties = {
+const getDimTextStyle = (isLeftColumn: boolean, clampCount: number): React.CSSProperties => ({
   ...textStyle,
+  whiteSpace: "unset",
   fontWeight: "bold",
   marginLeft: 4,
-};
+  overflow: "hidden",
+  textOverflow: "ellipsis",
+  display: "-webkit-box",
+  ...(!isLeftColumn && {
+    lineClamp: clampCount,
```

Comment:
Does the `line-clamp` actual exist? Cannot find it anywhere on MDN

Reply:
My assumption was passing it there + vendor prefix one with webkit (since apparently firefox supports it as well) and when it becomes stable (without vendor prefix) we are going to already have it there. 

But now, I think probably we don't need it. will remove it and only keep webkit one!